daint/.bashrc

```diff
@@ -109,36 +109,36 @@ if ! shopt -oq posix; then
 fi
 
 # nvim
-alias vim='/users/subbiali/neovim_build/bin/nvim'
-export VIM=/users/subbiali/neovim_build/bin/nvim
-export VIMRUNTIME=/users/subbiali/neovim_build/share/nvim/runtime
+alias vim='/users/"$USER"/neovim_build/bin/nvim'
+export VIM=/users/"$USER"/neovim_build/bin/nvim
+export VIMRUNTIME=/users/"$USER"/neovim_build/share/nvim/runtime
 
 # path
-export PATH="/users/subbiali/go1.19.2/bin:/users/subbiali/gitlfs_build/bin:$PATH"
+export PATH="/users/"$USER"/go1.19.2/bin:/users/"$USER"/gitlfs_build/bin:$PATH"
 
 # locale
 export LC_ALL=C.UTF-8
 export LANG=C.UTF-8
 
 # project
-export PROJECT=/project/s299/subbiali
+export PROJECT=/project/s299/"$USER"
```

Comment:
The upper-level folder `s299` needs to be parameterized too, e.g.:
```suggestion
project_id=$(sacct --format=Account --noheader | head -n 1 | awk '{$1=$1}1')
export PROJECT=/project/"$project_id"/"$USER"
```

Reply:
Thanks, I made changes respectively.